contents/blog/posthog-vs-statsig.mdx

```diff
@@ -0,0 +1,211 @@
+---
+title: "In-depth: PostHog vs Statsig"
+date: 2023-12-06
+author: ["lior-neu-ner"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: General
+tags:
+    - Comparisons
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonHeader } from 'components/ComparisonTable/header'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+PostHog and Statsig both offer A/B testing and feature flags, but they're different in two important ways:
+
+- **Statsig** offers advanced statistical methods for running tests. It lets you choose between Bayesian and Frequentist engines, run multi-armed bandit tests, and supports Bonferroni correction. It's ideal for data scientists and growth teams who need to conduct complex A/B testing.
+
+- **PostHog** is an all-in-one platform built to help engineers understand users and create better products. Beyond [A/B testing](/ab-testing) and [feature flags](/feature-flags), it offers advanced [product analytics](/docs/product-analytics), [session replays](/docs/session-replay), [surveys](/docs/surveys), and more.
+
+This post compares their platforms, experimentation features, pricing, and more.
+
+## How is PostHog different?
+
+### 1. It is an all-in-one platform
+
+PostHog brings together all the tools engineers need for testing, releasing, and measuring the success of new features. Feature flags and A/B testing are only part of a suite of tools PostHog offers.
+
+![PostHog homepage features](../images/blog/posthog-vs-statsig/all.png)
+
+PostHog combines usage, performance, and behavioral data with flags and experiments. PostHog’s [data warehouse](/docs/data-warehouse) also enables you to pull in data from external sources.
+
+Having all these product and data tools together enables you to do better analysis of shipped features and make better decisions about what you are building next. 
+
+### 2. PostHog is open source and transparent
+
+PostHog is built with transparency at its core. Not only do we work in the open and give full access to [our source code](https://github.com/PostHog/posthog), we also enable others to [build integrations](/docs/apps/build) or [other services on top of PostHog](/blog/how-we-built-an-app-server), open their own PRs, or give feedback on [our roadmap](/roadmap). PostHog's open app framework makes it easy to integrate internal tools, an advantage closed-source products like Statsig can't offer.
+
+### 3. PostHog constantly ships new features
+
+At PostHog, we ship weirdly fast. We update [our changelog](https://posthog.com/blog/posthog-changelog) with a recap of new features every week, and often there’s even more in beta testing. And that’s without counting all the apps and integrations submitted by our community! We work hard to keep PostHog on the cutting edge and we’re transparent about the ways we do that.
+
```

Comment:
Would love feedback on this. They do also publish a [changelog](https://statsig.com/updates), but given that they're also dev-focused, I wasn't sure what else to put.